### PR TITLE
Qa/doc test plan

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers
@flopezag @hmunfru 

#### Description
UPDATE Test project documentation and Test Plan for release 4.1

### CODE REVIEW

File: test/acceptance/README.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+If you need more information about how to use Vagrant, you can see
+`Vagrant Getting Started <https://docs.vagrantup.com/v2/getting-started/index.html>`_
+
+
+Test structure and prerequisites
+---------------------------------
+
+Acceptance tests have two type of test cases:
+
+- **Component test cases**: They try to test Sagitta catalog and they needn't additional prerequisites.
```
Comment (flopezag) [inline test/acceptance/README.rst]: they do not need

**Comment by hmunfru** [comment]
LGTM

**Comment by jframos** [comment]
ping @flopezag

**Comment by flopezag** [comment]
LGTM

**Pull request merged**